structure/0018-technical-teams.md

```diff
@@ -35,12 +35,12 @@ The following is a list of top-level technical teams.
-| Quilt Standard Libraries | [Quilt Standard Libraries](https://github.com/QuiltMC/quilt-standard-libraries)<sup>TBD</sup>| Will contain additional teams for each library
-| Infrastructure | [Quilt Meta](https://github.com/QuiltMC/quilt-meta)
-| Mappings | [Yarn](https://github.com/QuiltMC/yarn)<br>[Intermediary](https://github.com/QuiltMC/intermediary)<br>[Matcher](https://github.com/QuiltMC/matcher)<br>[Matcher MC](https://github.com/QuiltMC/matcher-mc)<br>[Enigma](https://github.com/QuiltMC/yarn)<br>[Lorenz Tiny](https://github.com/QuiltMC/lorenz-tiny)<br>[Stitch](https://github.com/QuiltMC/stitch)<br>[Tiny Mappings Parser](https://github.com/QuiltMC/tiny-mappings-parser)<br>[Tiny Remapper](https://github.com/QuiltMC/tiny-remapper)
-| Build Tools | [Quilt Loom](https://github.com/QuiltMC/quilt-loom)<br>[Gradle Convention Plugins](https://github.com/QuiltMC/gradle-convention-plugins)<br>[Sponge Mixin Compile Extensions](https://github.com/QuiltMC/sponge-mixin-compile-extensions)<br>[Dev Launch Injector](https://github.com/QuiltMC/dev-launch-injector)| This team will also be responsible for developing a replacement for Quilt Loom when it becomes possible to do so
+| Quilt Standard Libraries | [Quilt Standard Libraries](https://github.com/QuiltMC/quilt-standard-libraries) (+ separate triage team)<br>[Fabric API QSL](https://github.com/QuiltMC/fabric-api-qsl)<br>Contains several subteams:<ul><li>Block</li><li>Command</li><li>Core</li><li>Data</li><li>Entity</li><li>GUI</li><li>Item</li><li>Rendering</li><li>Transfer</li><li>Worldgen</li></ul>
+| Infrastructure | [Quilt Meta](https://github.com/QuiltMC/quilt-meta)<br>[LauncherMeta Parser](https://github.com/QuiltMC/launchermeta-parser)
+| Mappings | [Quilt Mappings](https://github.com/QuiltMC/quilt-mappings) (+ separate triage team)<br>[Mappings Hasher](https://github.com/QuiltMC/mappings-hasher)<br>[Enigma](https://github.com/QuiltMC/enigma)<br>[Lorenz Tiny](https://github.com/QuiltMC/lorenz-tiny)<br>[Stitch](https://github.com/QuiltMC/stitch)<br>[Tiny Mappings Parser](https://github.com/QuiltMC/tiny-mappings-parser)<br>[Tiny Remapper](https://github.com/QuiltMC/tiny-remapper)<br>[Mapping Poet](https://github.com/QuiltMC/mapping-poet)<br>[Quilt Mappings on Loom](https://github.com/QuiltMC/quilt-mappings-on-loom)<br>[Unpick](https://github.com/QuiltMC/unpick)
+| Build Tools | [Quilt Loom](https://github.com/QuiltMC/quilt-loom)<br>[Gradle Convention Plugins](https://github.com/QuiltMC/gradle-convention-plugins)<br>[Sponge Mixin Compile Extensions](https://github.com/QuiltMC/sponge-mixin-compile-extensions)<br>[Dev Launch Injector](https://github.com/QuiltMC/dev-launch-injector)<br>[Javadoc Draftsman](https://github.com/QuiltMC/javadoc-draftsman) | This team will also be responsible for developing a replacement for Quilt Loom when it becomes possible to do so
```

Comment:
Draftsman is a mappings team tool

Reply:
Fixed in a16d7b45ffa49f6ec8773969370109ae9a55e7c0, looked more like a build tool thing to me but that makes sense